diff --git a/dist/types/helper/adapter/index.d.ts b/dist/types/helper/adapter/index.d.ts
index 21f743cd0260447fe900f4ced599c8e5ec163ca0..d5156cc955fbc1e82b0638c4b4f0d0a3b78ab0af 100644
--- a/dist/types/helper/adapter/index.d.ts
+++ b/dist/types/helper/adapter/index.d.ts
@@ -4,7 +4,7 @@
  */
 import type { Context } from '../../context';
 export type Runtime = "node" | "deno" | "bun" | "workerd" | "fastly" | "edge-light" | "other";
-export declare const env: <T extends Record<string, unknown>, C extends Context = Context<{}, any, {}>>(c: C, runtime?: Runtime) => T & C["Env"];
+export declare const env: <T extends Record<string, unknown>, C extends Context = Context<{}, any, {}>>(c: C, runtime?: Runtime) => T & C["env"];
 export declare const knownUserAgents: Partial<Record<Runtime, string>>;
 export declare const getRuntimeKey: () => Runtime;
 export declare const checkUserAgentEquals: (platform: string) => boolean;
